.gitignore

```diff
@@ -38,17 +38,10 @@ captures/
 
 # IntelliJ
 *.iml
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/gradle.xml
-.idea/assetWizardSettings.xml
-.idea/dictionaries
-.idea/libraries
-# Android Studio 3 in .gitignore file.
-.idea/caches
-.idea/modules.xml
-# Comment next line if keeping position of elements in Navigation Editor is relevant for you
-.idea/navEditor.xml
+/.idea/*
+!/.idea/codeInsightSettings.xml
+!/.idea/codeStyles/
+!/.idea/inspectionProfiles/
```

Comment:
That's a good point, I would have to think more about it. Currently, I would want to avoid all idea files and build files being pushed. As done almost everywhere, as seen in  
[coil](https://github.com/coil-kt/coil/blob/main/.gitignore) as well